server/ai_http.go

```diff
@@ -633,7 +650,7 @@ func parseMultiPartResult(body io.Reader, boundary string, pipeline string) core
 		if p.Header.Get("Content-Type") == "application/json" {
 			var results interface{}
 			switch pipeline {
-			case "text-to-image", "image-to-image", "upscale", "image-to-video":
+			case "text-to-image", "image-to-image", "upscale", "image-to-video", "object-detection":
```

Comment:
I will rebuild and run E2E tomorrow but this needs to be changed to sending back the full ObjectDetection response.

Reply:
Thanks for pointing this one out, was missed in the ObjectDetectionResponse commit :+1: 

Done in the latest commit :+1: